data/NBO/Trp_polar.nbo.out

```diff
@@ -0,0 +1,1032 @@
+
+ *********************************** NBO 7.0 ***********************************
+             N A T U R A L   A T O M I C   O R B I T A L   A N D
+          N A T U R A L   B O N D   O R B I T A L   A N A L Y S I S
+ *************************** Eric Berquist (I102531) ***************************
```

Comment:
Probably not important but good to keep in mind.  (See [Q27](https://nbo6.chem.wisc.edu/faq_css.htm).)  I think this usage is ok.